includes/class-edd-download.php

```diff
@@ -101,6 +101,17 @@ class EDD_Download {
 	private $button_behavior;
 
 	/**
+	 * Default download support options
+	 *
+	 * @since 2.4
+	 */
+	private $supports = array(
```

Comment:
I was planning to add the checks for each of the builtin features in the support method once we nailed down functionality. So this works for me. Do you simply want to remove the $supports array entirely?